azurerm/internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -89,7 +89,7 @@ func resourceArmLogAnalyticsWorkspace() *schema.Resource {
 			"retention_in_days": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				Computed:     true,
+				Default:      30,
```

Comment:
I think this needs to remain Computed and not defaulted to handle the use-case for `7` which is still valid?